content/10.diagnostics.md

```diff
@@ -82,14 +82,46 @@ Although these approaches are also PCR based, they allow for more rapid scaling
+In their assay, guide RNAs were designed to recognize portions of sequences corresponding to the SARS-CoV-2 genome, specifically the N2 and E regions [@doi:10.1038/s41587-020-0513-4].
 In the presence of SARS-CoV-2 genetic material, sequence recognition by the guide RNAs results in double-stranded DNA cleavage by Cas12, as well as cleavage of a single-stranded DNA molecular beacon.
-The cleavage of this molecular beacon acts as a colorimetric reporter that is subsequently read out in a lateral flow assay and indicates the positive presence of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection [@doi:10.1101/2020.03.06.20032334].
-This assay has been reported to have a sensitivity as high as detection of 70-300 copies of the target RNA/µl, requires simple, accessible equipment, and can output an easily-interpretable result in approximately 30 minutes [@doi:10.1126/science.abb8400].
-Initial testing with patient samples (n = 23) demonstrated a positive predictive value of 100% and a negative predictive value of 91.7%, which is highly competitive with the CDC's current testing standard using qRT-PCR.
-This test seeks to develop a practical solution for rapid, low-barrier testing in areas that are at greater risk of infection, such as airports and local community hospitals.
+The cleavage of this molecular beacon acts as a colorimetric reporter that is subsequently read out in a lateral flow assay and indicates the positive presence of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection.
+The 40-minute assay is considered positive if there is detection of both the E and N genes or presumptive positive if there is detection of either of them.
+The assay had 95% positive predictive agreement and 100% negative predictive agreement with the US Centers for Disease Control and Prevention SARS-CoV-2 real-time RT–PCR assay.
+The estimated limit of detection was 10 copies per μl reaction, versus 1 copy per μl reaction for the CDC assay.
+These results have been confirmed by other DETECTR approaches.
+Using RTRPA for amplification, another group detected 10 copies of synthetic SARS-CoV-2 RNA per μl of input within 60 minutes of RNA sample preparation in a proof-of-principle evaluation [@doi:10.1101/2020.02.29.971127].
+The DETECTR protocol was improved by combining RT-RPA and CRISPR-based detection in a one-pot reaction that incubates at a single temperature, and by using dual crRNAs (which increases sensitivity).
+This AIODCRISPR assay (All-In-One Dual CRISPR-Cas12a) detected 4.6 copies of SARS-CoV-2 RNA per μl of input in 40 minutes [@doi:10.1101/2020.03.19.998724].
+Another single-tube, constant-temperature approach using Cas12b instead of Cas12a achieved a detection limit of 5 copies/μl in 40-60 minutes [@doi:10.1101/2020.04.10.023358].
```

Comment:
```suggestion
Another single-tube, constant-temperature approach using Cas12b instead of Cas12a achieved a detection limit of 5 copies/&mu;l in 40-60 minutes [@doi:10.1101/2020.04.10.023358].
```